registry/storage/driver/azure/azure.go

```diff
@@ -118,7 +119,9 @@ func (d *driver) GetContent(ctx context.Context, path string) ([]byte, error) {
 
 // PutContent stores the []byte content at a location designated by "path".
 func (d *driver) PutContent(ctx context.Context, path string, contents []byte) error {
-	if limit := 64 * 1024 * 1024; len(contents) > limit { // max size for block blobs uploaded via single "Put Blob"
+	// max size for block blobs uploaded via single "Put Blob" for version after "2016-05-31"
+	// https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob#remarks
+	if limit := 256 * 1024 * 1024; len(contents) > limit {
```

Comment:
[nit] I'd declare this as:

``` Go
const limit = 256 * 1024 * 1024
if len(contents) > limit {
    // stuff
}
```

Reply:
updated the const declaration. 

Regarding the new storage repo, how reliable is it? I saw only prelease builds for github.com/Azure/azure-storage-blob-go ( as well as the referenced package Azure/azure-pipeline-go). This is a bit uncomfortable. Would prefer a stable version. Do you think it's Ok to move to azure-storage-blob-go later once it starts to generate official release?